README.md

```diff
@@ -20,7 +20,8 @@ The container is configured via a set of required environment variables:
 - `S3_BUCKET_URL`: in most cases this should be `s3://name-of-your-bucket/`
 - `AWS_DEFAULT_REGION`: The AWS region your bucket resides in
 - `CRON_SCHEDULE`: Check out [crontab.guru](https://crontab.guru/) for some examples:
-- `BACKUP_NAME`: A name to identify your backup among the other files in your bucket, it will be postfixed with the current timestamp (date and time)
+- `BACKUP_NAME`: A name to identify your backup among the other files in your bucket 
+- `BACKUP_NAME_TIMESTAMP`: It will postfixed `BACKUP_NAME` with the current timestamp (date and time) (Optional, defaults to true)
```

Comment:
Maybe rephrase this into "`BACKUP_NAME` will be postfixed with the current timestamp (etc)" ?